src/plugin-slots/CourseAuthoringUnitSidebarSlot/README.md

```diff
@@ -1,18 +1,30 @@
 # CourseAuthoringUnitSidebarSlot
 
-### Slot ID: `org.openedx.frontend.authoring.course_unit_sidebar.v1`
+The component currently holds 2 versions of the slot. Version 2 wraps version 1 and hence, is a
+superset of version 1.
```

Comment:
Since the goal is to deprecate the `v1` version of this slot, I think it would make more sense to more cleanly split this documentation up instead of having info about both `v1` and `v2` in each section.

I think either having a `README.v1.md` file that `README.md` links to or having 2 top-level sections in the README would make it a lot easier to remove all the `v1` documentation when we're ready to remove the `v1` slot.

Reply:
I went with your first suggestion of adding a `README.v1.md` file.